### DISCUSSION TITLE
Debug cheats

### DISCUSSION THREAD

**Discussion by roniK23** [nroutasuo/level13#276] [Q&A]
Debug cheats were mentioned in the changelog. Where can I access them?

**Comment by nroutasuo** [comment]
If you mean the tools page from the changelog for version 0.3.2, there are currently none active but the page is here: [tools.html](https://nroutasuo.github.io/level13/tools.html)

If you mean some cheats mentioned in the git commits, then those are only there to make development and testing easier, and are available if you run a local debug version of the game.
